pyramid_swagger/api.py

```diff
@@ -101,6 +101,29 @@ def view_for_api_declaration(request):
     return view_for_api_declaration
 
 
+def resolve_ref(spec, url):
```

Comment:
Can we adjust the configuration.rst page accordingly to make sure we well-explain when and why to use this?